src/services/lnd/index.ts

```diff
@@ -305,28 +305,34 @@ const lookupPaymentByPubkeyAndHash = async ({
       : PaymentStatus.Pending
 
     let paymentLookup: LnPaymentLookup = {
-      amount: toSats(0),
-      createdAt: new Date(0),
```

Comment:
I changed this here to default to `undefined` instead because 1970 dates in the db was looking confusing for null values. Is there any reason why we initially put this to be `new Date(0)` @dolcalmi?

Reply:
yes, created_at is not optional https://github.com/alexbosworth/ln-service#getpayment

Reply:
Right, but the entire `payment` object is optional so there's cases where this isn't present